i18n/fr_FR/LC_MESSAGES/Alltube.po

```diff
@@ -221,15 +221,15 @@ msgstr "Impossible de trouver l'URL de la vidéo."
 msgid ""
 "Easily download videos from Youtube, Dailymotion, Vimeo and other websites."
 msgstr ""
-"Téléchargez facilement des vidéos depuis Youtube, Dailymotion, Vimeo et "
+"Téléchargez facilement des vidéos depuis YouTube, Dailymotion, Vimeo et "
 "d'autres sites web."
 
 #: controllers/FrontController.php:110
 msgid ""
-"List of all supported websites from which Alltube Download can extract video "
+"List of all supported websites from which AllTube Download can extract video "
```

Comment:
If you edit the source string, you should change it in `FrontController.php` as well.